garage/tf/models/gaussian_mlp_model.py

```diff
@@ -0,0 +1,227 @@
+"""Gaussian MLP Model."""
+import numpy as np
+import tensorflow as tf
+
+from garage.core import Serializable
+from garage.misc import ext
+from garage.misc.overrides import overrides
+from garage.tf.core.mlp import mlp
+from garage.tf.core.parameter import parameter
+from garage.tf.models import Model
+
+
+class GaussianMLPModel(Model, Serializable):
```

Comment:
does this behave properly if I use `tf.keras.Model` APIs like `to_json()` `weights` ,etc?

Reply:
still waiting on an answer to this question

Reply:
No. From Keras documentation, 
```
In subclassed models, the model's topology is defined as Python code (rather than as a static graph of layers). That means the model's topology cannot be inspected or serialized. As a result, the following methods and attributes are not available for subclassed models:

model.inputs and model.outputs.
model.to_yaml() and model.to_json()
model.get_config() and model.save().
```
`model.get_weights()` and `model.set_weights(weights)` are still available, however they return a empty list, so I guess subclassed models will have to implement them as well.

Reply:
or are we actually using the primitives, e.g. mlp, from keras?

Reply:
If we want to use `keras.models.Model` and comply with keras API, we may also want to use their primitives (e.g. mlp) so that we reuse their properties (e.g. get_config(), to_json(), get_weights()). For custom model (e.g. GaussianMLPModel), we can also write it in `keras.layers.core` (e.g. Dense, Activation). 

Otherwise, we would have to write those functions ourselves and it seems we don't get significant benefit from following `keras.models.Model`. In this case, why don't we just use our own model class? (garage.Model)